workflow-templates/shellcheck.yml

```diff
@@ -0,0 +1,40 @@
+# Run ShellCheck on PRs and $default-branch
+
+# For more context, see:
+# https://github.com/openedx/.github/blob/master/docs/decisions/0001-shellcheck.rst
+
+name: ShellCheck
+
+on:
+  pull_request:
+  push:
+    branches:
+    - $default-branch
+
+permissions:
+  contents: read
+
+jobs:
+  shellcheck:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out this repository
+      uses: actions/checkout@v3
+
+    - name: Check out shared actions
+      uses: actions/checkout@v3
+      with:
+        repository: openedx/.github
+        path: ./openedx-dot-github
```

Comment:
Apparently the only way to re-use actions is to (a) publish them or (b) literally clone their repo and use the action 'locally'. Publishing the action sounds like a pain, so I'm trying out (b).

Reply:
Outdated since I switched to a shared workflow.